src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1270,6 +1274,55 @@ export class ResultsViewPageStore {
         },
     });
 
+    // Isolate discrete CNA data from other NumericMolecularData
+    // and add the custom driver annotations to data points
+    readonly discreteCNAMolecularData = remoteData<
```

Comment:
to keep with convention, I think this should be `annotatedDiscreteCNAMolecularData`